src/context/userDataContext.js

```diff
@@ -0,0 +1,29 @@
+import { createContext, useReducer, useEffect, useContext } from "react";
+import { userDataReducer } from "../reducer/userDataReducer";
+import { useAuth } from "./authContext";
+import { getLikesService } from "../services/LikesServices";
+import { getWatchLaterService } from "../services/WatchLaterServices";
```

Comment:
You can export all services from a single index.js file inside the services folder and import them here from that file